src/Tribe/Context.php

```diff
@@ -1160,7 +1144,7 @@ public function to_array(  ) {
 	 *
 	 * @return array
 	 */
-	public function get_state( array $fields = null, $whitelist = true ) {
+	public function get_state( ?array $fields = null, $whitelist = true ) {
```

Comment:
Here and elsewhere.
Since you're here: add the `array` return type and document the `@return array` value.